config/default.yml

```diff
@@ -0,0 +1,4 @@
+Minitest/AssertNil:
+  Description: 'Check if your test uses assert_nil instead of assert_equal(nil, something).'
```

Comment:
Enclose the code with backticks.

```diff
-'Check if your test uses assert_nil instead of assert_equal(nil, something).'
+'Check if your test uses `assert_nil` instead of `assert_equal(nil, something)`.'
```